### DISCUSSION TITLE
[已解决]高德地图无法搜索

### DISCUSSION THREAD

**Discussion by ZeR0Ke1th** [blackmatrix7/ios_rule_script#1032] [General]
两周前开始发现高德地图无法搜索地点，且有一部分界面进入后会弹出异常，望大佬修复！
![微信图片_20230929111432](https://github.com/blackmatrix7/ios_rule_script/assets/20182767/4f1d63fc-ebc1-4830-92fe-22478cc76cd8)
![微信图片_20230929111444](https://github.com/blackmatrix7/ios_rule_script/assets/20182767/c9c14005-4f45-4437-b2dd-da3d1b23b031)

**Comment by donf6938** [comment]
+1
